subprojects/snapshots/src/main/java/org/gradle/internal/fingerprint/DirectorySensitivity.java

```diff
@@ -33,7 +33,18 @@
     /**
```

Comment:
`DEFAULT` is not `CONSIDER_DIRECTORIES`. For example, `DEFAULT` in combination with classpath normalization means to ignore directories. Same in combination with ignore path normalization. So I'd say we keep `DEFAULT` here since it means the default for the chosen normalization strategy.

Reply:
Ah, yes, you're right. I still don't like `DEFAULT`, as it implies a lack of more explicit specification (which is also what `UNSPECIFIED` does). `INHERIT` comes to mind, but I don't like that either...

Another option would be to instead have a `CONSIDER_DIRECTORIES` value for `DirectorySensitivity`, and have `@CompileClasspath` _imply_ `IGNORE_DIRECTORIES`. That would be cleaner for me. WDYT?

Reply:
I'd like to have something like this, so have the strategy specify whether there is a version with or without directory sensitivity and then being able to query the result for the directory sensitivity. Though I'd prefer not to include the change in this PR. I also wonder what the effects for build scans would be if we change the meaning of `DEFAULT`.